python/lsst/ap/association/diaPipe.py

```diff
@@ -493,6 +620,38 @@ def run(self,
                                         createResults.nNewDiaObjects,
                                         nTotalSsObjects,
                                         nAssociatedSsObjects)
+        return (associatedDiaSources, createResults.newDiaObjects)
```

Comment:
It would be helpful to add a log message just before the return:
```
        self.log.info("%i updated and %i unassociated diaObjects. Creating %i new diaObjects",
                      assocResults.nUpdatedDiaObjects,
                      assocResults.nUnassociatedDiaObjects,
                      createResults.nNewDiaObjects,
                      )
```